lib/service/TokenList.dart

```diff
@@ -78,7 +78,12 @@ class TokenList {
     ["Fernando Tatis Jr.", 10007217, "FTJLT1010", "FTJST1010"],
     ["Jose Ramirez", 10000352, "JRLT1010", "JRST1010"],
     ["Juan Soto", 10006794, "JSLT1010", "JSST1010"],
-    ["Marcus Semien", 10000908, "MSLT1010", "MSST1010",], 
+    [
+      "Marcus Semien",
+      10000908,
+      "MSLT1010",
+      "MSST1010",
+    ],
```

Comment:
why is this happening here? looks like nothing was changed but just formatted worse